### PR TITLE
chore(local): script for setting Predator locally

### PR DESCRIPTION
**Pull request by enudler**

### CODE REVIEW

**Comment by enudler** [comment]
Related issues: #416 

Also, upgrade eslint version and changed let to const everywhere it was an option

File: setupLocalEnv.sh [hunk 1]
```diff
@@ -0,0 +1,37 @@
+cat << "EOF"
```
Comment (syncush) [inline setupLocalEnv.sh:1]: we need to specify a shebang!

**Review by syncush — CHANGES_REQUESTED** [review]
1. Have you noticed that you pushed some binary files?
2. suggestion: we should rename our default sqlite file name to predator.sqlite and add `*.sqlite` to `.gitignore`, wdyt?
3. lets set a .nvmrc file
4. set engine in `package.json`
5. Add to README.md and docs website the URL for the predator dashboard

**Comment by enudler** [comment]
@syncush done
added grafana dashboard to our docs (it was before that on the faq)

**Review by syncush — APPROVED** [review]
Perfect

**Pull request merged**